pyproject.toml

```diff
@@ -9,11 +9,11 @@ package-mode = false
 
 [tool.poetry.dependencies]
 python = "^3.12.6"
-fastapi = "^0.116.1"
+fastapi = "^0.120.1"
 uvicorn = "^0.34.2"
 semver = "^3.0.4"
 jsonpath-ng = "^1.7.0"
-starlette = "^0.47.2"
+starlette = "^0.49.1"
```

Comment:
I don't think we need `starlette` listed here, funny enough, even, deptry is outputting: `DEP002 'starlette' defined as a dependency but not used in the codebase` but ok for now.